kratos/utilities/normal_calculation_utils.h

```diff
@@ -103,12 +103,14 @@ class KRATOS_API(KRATOS_CORE) NormalCalculationUtils
      * @brief It computes the mean of the normal in the entities and in all the nodes
      * @param rModelPart The model part to compute
      * @param EnforceGenericGeometryAlgorithm If enforce the generic algorithm for any kind of geometry
+     * @param ConsiderUnitNormal If consider the unit normal
```

Comment:
```suggestion
     * @param ConsiderUnitNormal In order to consider directly the unit normal instead of the area normal multiplied with a coefficient
```